docs/user/00-10-overview-istio-controller.md

```diff
@@ -31,4 +31,13 @@ However, if a resource is a job, a ReplicaSet that is not managed by any deploym
 
 ## X-Forwarded-For HTTP Header
 
-The **X-Forwarded-For** (XFF) header is only supported on AWS clusters.
\ No newline at end of file
+The **X-Forwarded-For** (XFF) header is only supported on AWS clusters.
+
+The X-Forwarded-For header conveys the client IP address and the chain of intermediary proxies traversed by the request to reach the Istio Service Mesh.
```

Comment:
```suggestion
The XFF header conveys the client IP address and the chain of intermediary proxies that the request traversed to reach the Istio service mesh.
```
the rule about abbreviations is that once we explain what it stands for ([like here](https://github.com/kyma-project/istio/pull/533/files#diff-e1e09ad28d658e440e67eeaab48533f8ec8b17e7e34315d3642e91d111aa268fR34)), we can use it freely